lib/galaxy/tool_util/parser/xml.py

```diff
@@ -612,7 +613,7 @@ def __parse_test_attributes(output_elem, attrib, parse_elements=False, parse_dis
     # Number of lines to allow to vary in logs (for dates, etc)
     attributes['lines_diff'] = int(attrib.pop('lines_diff', '0'))
     # Allow a file size to vary if sim_size compare
-    attributes['delta'] = float(attrib.pop('delta', '10000'))
+    attributes['delta'] = float(attrib.pop('delta', DEFAULT_DELTA))
```

Comment:
Can you also replace the equivalent part in yaml.py ?

Reply:
Excellent idea, in particular since yet another default was used there.